plugins/usbdmx/LibUsbThreadTest.cpp

```diff
@@ -87,6 +87,10 @@ void LibUsbThreadTest::testNonHotplug() {
 
 #if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000102)
 void LibUsbThreadTest::testHotplug() {
+  if (!m_context) {
+    return;
```

Comment:
There are other reasons for this not running, like the #if checks. I don't want to get into the pattern of adding logs for all of these.